cob_hardware_config/cob4-2/config/teleop.yaml

```diff
@@ -72,7 +72,7 @@ components: {
   head: {
     sss_default_target: "front",
     velocity_topic_name: "/head/joint_group_velocity_controller/command",
-    joint_velocity: [0.05, 0.05],
+    joint_velocity: [0,05, 0.05, 0.05],
```

Comment:
if a robot does't have a head , the component should not be included , for example for raw3-1: https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_hardware_config/raw3-1/config/teleop.yaml 

Teleop will find all the components listed under` components: { ` and allow all the modes defined (` joy_num_modes:` ) for all the listed components.

For example for cob4-2 ( https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_hardware_config/cob4-2/config/teleop.yaml ) you have a list of 6 components (base, torso, head, sensorring, arm_left, arm_right), the parameter joy_num_modes is 4, and the button functions (parameters *_button) are also defined for 4 modes. 

that means that you can initialize:

- base, torso, head, sensorring, arm_left, arm_right

and you can move:

1. the base-twist command 
2. the torso, head, sensorring, arm_left and arm_right to a default position with a trajectory controller
3. the torso, head, sensorring, arm_left and arm_right  using the velocity controller
4. the torso, head, arm_left and arm_right using the twist controller.


For example for raw3-1 ( https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_hardware_config/raw3-1/config/teleop.yaml ) you have a list of 1 components (base),  the parameter joy_num_modes is 1, and the button functions (parameters *_button) is only for mode1 defined:

that means that you can initialize:

- base

and you can move:

1. the base-twist command 

if the `head_sss_default_target_button` is defined and you didn't include "head" as component or you only have one mode , the parameter will be just not used